src/engine/menus/config.asm

```diff
@@ -5,7 +5,7 @@ _PauseMenu_Config:
 	push af
 	xor a
 	ld [wConfigExitSettingsCursorPos], a
-	ld a, 1
+	ld a, $01 ; text isn't double-spaced
```

Comment:
Rather than changing this instance to `$01`, I would rather change most other instances to `1`. Or use a constant. It would be tempting to use `TRUE` but that would feel inverted from the real meaning based on the current name of `wLineSeparation`. If it were named like eg `wSingleSpaced` then `TRUE` would feel more natural. Or we could leave the name alone and make a special `SINGLE_SPACED` constant instead of using `TRUE`. Just rambling..

Reply:
I only added those "double-spaced" comments to try to make the updates to that variable easier to understand. I also agree that either of your suggestions would probably be a better idea than my comments. Just tell me what you would prefer, and I'll update the pull request.

Reply:
My vote would probably be for adding:

```asm
DEF DOUBLE_SPACED EQU 0
DEF SINGLE_SPACED EQU 1
```

and then doing a bunch of:

```diff
- 	xor a ; text is double-spaced
+ 	xor a ; DOUBLE_SPACED
```

and

```diff
- 	ld a, $01 ; text isn't double-spaced
+ 	ld a, SINGLE_SPACED
```